ariadne/src/Ariadne/Wallet/Cardano/Kernel/DB/HdWallet/Read.hs

```diff
@@ -72,12 +72,11 @@ check f g = using' f (const g)
 
 -- | Current balance of the entire root
 hdRootBalance :: HdRootId -> HdQuery Integer
-hdRootBalance rootId =
-    sumCoins
-    . map hdAddressBalance
-    . toList
-    . IxSet.getEQ rootId
-    . view hdWalletsAddresses
+hdRootBalance rootId = sumCoins
+                     . map hdAccountBalance
+                     . toList
+                     . IxSet.getEQ rootId
+                     . view hdWalletsAccounts
```

Comment:
It's better to squash this commit with the one where it was introduced.

Reply:
I will squash everything into a single commit when I merge.

Reply:
Well, anyway, it's better to squash before requesting review, because some people review per commit.